framework/wazuh/agent.py

```diff
@@ -123,6 +123,27 @@ def restart_agents(agent_list=None):
     return result
 
 
+@expose_resources(actions=['cluster:read_config'], resources=['node:id:{node_id}'], post_proc_func=None)
+def restart_agents_by_node(agent_list=None, node_id=None):
+    """Restart all agents belonging to a node.
+
+    Parameters
+    ----------
+    agent_list : list, optional
+        List of agents. Default `None`
+    node_id : str, optional
+        Node name. Only used for RBAC. Default `None`
+
+    Returns
+    -------
+    AffectedItemsWazuhResult
+    """
+    if '000' in agent_list:
```

Comment:
This error will appear otherwise and we cannot use the `post_proc_func`:
```
{
        "error": {
          "code": 1703,
          "message": "Action not available for Manager (agent 000)",
          "remediation": "Please, use `GET /agents?select=id,name` to find all available agents and make sure you select an agent other than 000"
        },
        "id": [
          "000"
        ]
      },
```